### PR TITLE
Docs revamp

### PR DESCRIPTION
**Pull request by NicoZelaya**
https://splitio.atlassian.net/browse/SDKS-910

### CODE REVIEW

File: CONTRIBUTORS-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+5. Run the build and make sure it runs with no errors.
+6. Run all tests and make sure there are no failures.
+7. `git push` your changes to GitHub within your topic branch.
+8. Open a Pull Request(PR) from your forked repo and into the `development` branch of the original repository.
+9. When creating your PR, please fill out all the fields of the PR template, as applicable, for the project.
+10. Check for conflicts once the pull request is created to make sure your PR can be merged cleanly into `development`.
+11. Keep an eye out for any feedback or comments from Split's SDK team.
+  
+### Building the SDK
+Instructions on how to build this library
```
Comment (NicoZelaya) [inline CONTRIBUTORS-GUIDE.md]: @javrudsky Add instructions and remove placeholder

File: CONTRIBUTORS-GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  
+### Building the SDK
+Instructions on how to build this library  
+
+ADD CONTENT
+   
+### Running tests
+Instructions on how to run automated tests
+
+ADD CONTENT
```
Comment (NicoZelaya) [inline CONTRIBUTORS-GUIDE.md]: @javrudsky Add instructions and remove placeholders

File: CONTRIBUTORS-GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  
+### Linting and other useful checks
+Instructions on how to run the static code analysis tool of the project, as well as any other checks you may have available.
+
+ADD CONTENT
```
Comment (NicoZelaya) [inline CONTRIBUTORS-GUIDE.md]: @javrudsky Add instructions and remove placeholders

File: README.md [hunk 4]
```diff
@@ -1,42 +1,84 @@
...
+This SDK is compatible with Android SDK versions 15 and later (4.0.3 Ice Cream Sandwich).
+
+## Getting started
+Below is a simple example that describes the instantiation and most basic usage of our SDK:
+```
+<Factory instantiation>
+<Block until Ready>
+<Get treatment>
+<If treatment == ...>
+```
```
Comment (NicoZelaya) [inline README.md]: Same as for iOS, add the simplest code possible to "validate" the SDK behaviour. Use the official docs as inspiration
https://help.split.io/hc/en-us/articles/360020343291-Android-SDK#2-instantiate-the-sdk-and-create-a-new-split-client
https://help.split.io/hc/en-us/articles/360020343291-Android-SDK#basic-usage

File: CHANGES.txt [hunk 5]
```diff
@@ -1,59 +1,59 @@
 2.4.4: (Sep 25, 2019)
-- Fixed issue when load events from disk
+ - Bugfixing - issue when load events from disk
 2.4.3: (Sep 20, 2019)
```
Comment (EmilianoSanchez) [inline CHANGES.txt]: Add newline among releases

File: CONTRIBUTORS-GUIDE.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            Log.i("TAG", "CONTROL was returned, there was an error");
+        }
+    }
+
+    public void onPostExecutionView(SplitClient client) {
+        Log.i("TAG", "Do some UI work");
+    }
+
+});
+```
```
Comment (EmilianoSanchez) [inline CONTRIBUTORS-GUIDE.md]: this should go in README.md (basic usage of the SDK), in the placeholder:
```
<Factory instantiation>
<Block until Ready>
<Get treatment>
<If treatment == ...>
```

**Pull request merged**